### PR TITLE
chore(ci): improve workflows

### PR DESCRIPTION
**Pull request by manuelmeister**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/JsDaddy/ngx-mask/blob/develop/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Test only run on PRs and status only shows travis which is not used anymore.

## What is the new behavior?
Show the status of the develop branch and run CI on push.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by manuelmeister** [comment]
@NepipenkoIgor as you can see the build failing (without any modifications), it would be very important to enable the CI on the build branch.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -12,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [16.x]
```
Comment (manuelmeister) [inline .github/workflows/main.yml:17]: This should probably be the same as defined in package.json.
https://github.com/JsDaddy/ngx-mask/blob/b9968ae7cf27616678c487c67daa8de8088fe9a3/package.json#L6-L8

File: README.md [hunk 2]
```diff
@@ -2,17 +2,15 @@
   <h1 align="center">ngx-mask</h1>
 </a>
 
-[![Build Status](https://img.shields.io/travis/JsDaddy/ngx-mask.svg?branch=develop)](https://travis-ci.org/JsDaddy/ngx-mask)
+[![CI](https://github.com/JsDaddy/ngx-mask/actions/workflows/main.yml/badge.svg?branch=develop)](https://github.com/JsDaddy/ngx-mask/actions/workflows/main.yml)
```
Comment (manuelmeister) [inline README.md:5]: Travis hasn't run since 5 months

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
-name: Mask CI
+name: CI
 
 on:
+  push:
```
Comment (manuelmeister) [inline .github/workflows/main.yml:4]: Problems like this:
```
Error: projects/ngx-mask-lib/src/test/basic-logic.spec.ts:352:5 - error TS2322: Type 'null' is not assignable to type 'string'.

352     component.mask = null;
        ~~~~~~~~~~~~~~
```
could be detected much easier if we turn on the CI on push (meaning if something is pushed without PR)

**Comment by NepipenkoIgor** [comment]
@manuelmeister  Hi. Could you fix errors ?  And we will be able to merge and than will able to merge another PRs

**Comment by manuelmeister** [comment]
Should I remove the test where we set the mask to `null`? Because I don't understand why the mask can't be null in the TestMaskComponent.

**Comment by manuelmeister** [comment]
@NepipenkoIgor Should I remove the test where we set the mask to `null`? Because I don't understand why the mask can't be null in the TestMaskComponent.

**Comment by NepipenkoIgor** [comment]
@manuelmeister  Hi. Reverted and made update from scratch . Also was not be able to merge your PR. Copied your changes  in PR. Please check . Seems like test and CI works

**Pull request closed without merging**